.github/workflows/test.yml

```diff
@@ -51,3 +51,55 @@ jobs:
       uses: codecov/codecov-action@v1
       with:
         file: ./coverage.xml
+
+  build-win:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - uses: crazy-max/ghaction-chocolatey@v1
+      - name: Install curl
+        with:
+          args: install curl
+
+      - uses: crazy-max/ghaction-chocolatey@v1
+      - name: Install make
+        with:
+          args: install make
```

Comment:
Can we merge `curl` and `make` together?

Reply:
Done